### ISSUE TITLE
Awesome papers from Peter Bartlett's group

### ISSUE THREAD

**Issue by Chen-yu-Zheng** [WeiHuang05/Awesome-Feature-Learning-in-Deep-Learning-Thoery#1]
I think many insightful papers from Peter Bartlett's group can be added, and some of them are listed as follows.
1. Benign Overfitting in Linear Regression, PNAS, 2020
2. Benign overfitting without linearity: Neural network classifiers trained by gradient descent for noisy linear data, COLT, 2022
3. Implicit bias in leaky ReLU networks trained on high-dimensional data, ICLR, 2023
4. Random Feature Amplification: Feature Learning and Generalization in Neural Networks, JMLR, 2023
5. The Double-Edged Sword of Implicit Bias: Generalization vs. Robustness in ReLU Networks, NIPS, 2023
6. Benign overfitting in linear classifiers and leaky relu networks from kkt conditions for margin maximization, COLT, 2023

**Comment by WeiHuang05** [comment]
Thanks for your suggestion, I have included some of them in the reading list.
